portiaui/app/templates/components/project-listing.hbs

```diff
@@ -50,7 +50,7 @@
                   icon='vertical-ellipsis' as |options|
                 }}
                     {{#if versionControlled}}
-                        {{#if project.hasChanges}}
+                        {{#if (not hasNoChanges)}}
```

Comment:
Can you change the double negation into a `hasChanges` in the component by `hasChanges: computed.not('hasNoChanges')`?